client/signup/config/flows.js

```diff
@@ -257,7 +257,8 @@ function filterFlowName( flowName ) {
 
 	// Headstarted "default" flow (`newsite`) with vertical selection for EN users, coming from the homepage single button.
 	if ( 'website' === flowName && ( 'en' === locale || 'en-gb' === locale ) ) {
-		return 'newsite';
+		// Triforce test enabled for EN users, coming from the homepage single button.
```

Comment:
This should be cleaner and simpler with #4944 . It could be reduced to just this (like the test above it):

```php
if ( includes( defaultFlows, flowName ) && abtest( 'triforce' ) === 'triforce' ) {
	return 'layout';
}
```